src/Test/Perf/Tests/projects/csharp/CSharpCompilerNoAnalyzer.csx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
It would be better to have one test case running with each `.csx`. This way when one of the tests run by the `.csx` fails then it won't affect the other runs.

Also TraceManager when running the .csx file assumes that it is running one test case and the related files are generated in such fashion. 

So I think it is better to stick this change